looks like there's some failures;

```
10:05:29 =================================== FAILURES ===================================
10:05:29 __________________ BuildTest.test_build_container_with_target __________________
10:05:29 /docker-py/tests/integration/api_build_test.py:213: in test_build_container_with_target
10:05:29     self.assertEqual(info['Config']['OnBuild'], [])
10:05:29 E   AssertionError: None != []
10:05:29 ---------------------------- Captured stderr setup -----------------------------
10:05:29 
10:05:29 pulling busybox:buildroot-2014.02
10:05:29 Pulling from library/busybox - None
10:05:29 Pulling fs layer - None
10:05:29 Pulling fs layer - None
10:05:29 Downloading - [==================================================>]      32B/32B
10:05:29 Verifying Checksum - None
10:05:29 Download complete - None
10:05:29 Extracting - [==================================================>]      32B/32B
10:05:29 Downloading - [>                                                  ]  16.38kB/1.145MB
10:05:29 Extracting - [==================================================>]      32B/32B
10:05:29 Download complete - None
10:05:29 Pull complete - None
10:05:29 Extracting - [=>                                                 ]  32.77kB/1.145MB
10:05:29 Extracting - [=================>                                 ]  393.2kB/1.145MB
10:05:29 Extracting - [==================================================>]  1.145MB/1.145MB
10:05:29 Extracting - [==================================================>]  1.145MB/1.145MB
10:05:29 Pull complete - None
10:05:29 Digest: sha256:f8cdcf3eb8fac77a9b24f37bc440feea024a5d3ec9a33f1d0a4c42c5df6a49c3 - None
10:05:29 Status: Downloaded newer image for busybox:buildroot-2014.02 - None
10:05:29 ____________________ BuildTest.test_build_with_network_mode ____________________
10:05:29 /docker-py/tests/integration/api_build_test.py:231: in test_build_with_network_mode
10:05:29     assert self.client.inspect_image('dockerpytest_bridgebuild')
10:05:29 /docker-py/docker/utils/decorators.py:19: in wrapped
10:05:29     return f(self, resource_id, *args, **kwargs)
10:05:29 /docker-py/docker/api/image.py:262: in inspect_image
10:05:29     self._get(self._url("/images/{0}/json", image)), True
10:05:29 /docker-py/docker/api/client.py:226: in _result
10:05:29     self._raise_for_status(response)
10:05:29 /docker-py/docker/api/client.py:222: in _raise_for_status
10:05:29     raise create_api_error_from_http_exception(e)
10:05:29 /docker-py/docker/errors.py:31: in create_api_error_from_http_exception
10:05:29     raise cls(e, response=response, explanation=explanation)
10:05:29 E   ImageNotFound: 404 Client Error: Not Found ("no such image: dockerpytest_bridgebuild: No such image: dockerpytest_bridgebuild:latest")
```

/cc @shin-

--- a/Dockerfile
+++ b/Dockerfile
@@ -133,7 +133,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Get the "docker-py" source so we can run their integration tests
-ENV DOCKER_PY_COMMIT a962578e515185cf06506050b2200c0b81aa84ef
+ENV DOCKER_PY_COMMIT ca7a6132a418c32df6bb11ba9b2a8b9b2727227a
 # To run integration tests docker-pycreds is required.
 RUN git clone https://github.com/docker/docker-py.git /docker-py \
 	&& cd /docker-py \

--- a/Dockerfile.aarch64
+++ b/Dockerfile.aarch64
@@ -105,7 +105,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Get the "docker-py" source so we can run their integration tests
-ENV DOCKER_PY_COMMIT a962578e515185cf06506050b2200c0b81aa84ef
+ENV DOCKER_PY_COMMIT ca7a6132a418c32df6bb11ba9b2a8b9b2727227a
 # To run integration tests docker-pycreds is required.
 RUN git clone https://github.com/docker/docker-py.git /docker-py \
 	&& cd /docker-py \

--- a/Dockerfile.armhf
+++ b/Dockerfile.armhf
@@ -103,7 +103,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Get the "docker-py" source so we can run their integration tests
-ENV DOCKER_PY_COMMIT a962578e515185cf06506050b2200c0b81aa84ef
+ENV DOCKER_PY_COMMIT ca7a6132a418c32df6bb11ba9b2a8b9b2727227a
 # To run integration tests docker-pycreds is required.
 RUN git clone https://github.com/docker/docker-py.git /docker-py \
 	&& cd /docker-py \

--- a/Dockerfile.ppc64le
+++ b/Dockerfile.ppc64le
@@ -101,7 +101,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Get the "docker-py" source so we can run their integration tests
-ENV DOCKER_PY_COMMIT a962578e515185cf06506050b2200c0b81aa84ef
+ENV DOCKER_PY_COMMIT ca7a6132a418c32df6bb11ba9b2a8b9b2727227a
 # To run integration tests docker-pycreds is required.
 RUN git clone https://github.com/docker/docker-py.git /docker-py \
 	&& cd /docker-py \

--- a/Dockerfile.s390x
+++ b/Dockerfile.s390x
@@ -95,7 +95,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 # Get the "docker-py" source so we can run their integration tests
-ENV DOCKER_PY_COMMIT a962578e515185cf06506050b2200c0b81aa84ef
+ENV DOCKER_PY_COMMIT ca7a6132a418c32df6bb11ba9b2a8b9b2727227a
 # To run integration tests docker-pycreds is required.
 RUN git clone https://github.com/docker/docker-py.git /docker-py \
 	&& cd /docker-py \

--- a/hack/ci/janky
+++ b/hack/ci/janky
@@ -8,6 +8,6 @@ hack/test/unit
 hack/make.sh \
 	binary-daemon \
 	dynbinary \
-	test-integration \
 	test-docker-py \
+	test-integration \
 	cross

